packages/@lwc/integration-karma/test/api/freezeTemplate/index.spec.js

```diff
@@ -170,6 +170,8 @@ describe('freezeTemplate', () => {
         const stylesheet = () => 'div { color: red }';
         template.stylesheets = [stylesheet];
         template.stylesheetToken = 'myToken';
+        template.renderMode = undefined;
+        template.slots = undefined;
```

Comment:
We inspect the accessor descriptors later, but these are only patched in dev mode. It's easier to just set undefined here so it works the same in prod and dev.

The point of the test is just to confirm that people can still override these.